bluej/src/main/java/bluej/utility/javafx/DelegableScalableTextField.java

```diff
@@ -63,18 +72,40 @@ public class DelegableScalableTextField<DELEGATE_IDENT> extends ScalableHeightTe
 
     private final TextFieldDelegate<DELEGATE_IDENT> delegate;
     private final DELEGATE_IDENT delegateId;
+
+    // Keep track of InputMethod (IM) positions, e.g. for entering Chinese with
```

Comment:
This is very subtle and not critical for the code, but I would try to use another word than "positions", because at first, I thought it mean the "UI" position of the IME tool (like where it displays relative to the slot, or something of that nature).
Maybe just "InputMethod (IM) text's positions" clarifies that.